src/stories/Quiz/Hooks/index.stories.tsx

```diff
@@ -3,7 +3,7 @@ import MyQuiz, { addHookStoryCode } from './index';
 import '../../../styles.css';
 import { basicDescription, apiKey, quizId, hookDescription } from '../../../constants';
 import { stringifyWithDefaults } from '../../../utils';
-import { argTypes } from '../argTypes';
+import { argTypes, docsControls, storiesControls } from '../argTypes';
 
 export default {
   title: 'Quiz/Hooks',
```

Comment:
Not sure if this is the right place, but it seems like the control for `primaryColor` isn't working here. Tried without quotes, with quotes, with single quotes. 🤔 

Also, why do some of the descriptions have a `-` and some don't? I think they were showing that the type was in the Component Basic Usage story, but it seems like those aren't shown here.

![Screenshot 2023-10-05 at 5 25 20 PM](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/7194266/2caf1544-5208-409c-9a30-6ded7941fc9f)

Also, this is just me being anal retentive, but could we be a bit more consistent with the capitalizations here?
* `import` => `Import`
* `use` => `Use`
* And I'll leave capitalizing the sub-bullets (i.e. `state management`) to your best judgement

![Screenshot 2023-10-05 at 5 30 23 PM](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/7194266/9370cef4-0edb-4c4e-82d1-881e09adaf22)

Reply:
> Not sure if this is the right place, but it seems like the control for primaryColor isn't working here
The primary colors were not used in the hooks example. I added it in the example but it's not very user friendly.

> Also, why do some of the descriptions have a - and some don't? I think they were showing that the type was in the Component Basic Usage story, but it seems like those aren't shown here.
We have another PR for the hooks docs. But removed the `-` for now